inc/languages/english/admin/user_users.lang.php

```diff
@@ -91,6 +91,7 @@
 $l['error_remote_avatar_not_allowed'] = "Remote avatar URLs have been disabled by the forum administrator.";
 $l['error_invalid_user'] = "You have selected an invalid user.";
 $l['error_no_perms_super_admin'] = "You do not have permission to edit this user because you are not a super administrator.";
+$l['error_delete_self'] = "You cannot delete yourself.";
```

Comment:
```suggestion
$l['error_delete_self'] = "You are not allowed to delete your own account while logged in.";
```

Reply:
It's kind of weird to write "while logged in", it assumes you could perform the action whilst not logged in, which is similarly impossible.